secure/storage/src/tests/crypto_key.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
```

Comment:
I think these belong in the suite. Initially we might just use our implementation for vault and eventually move over to transit.

Reply:
Done, thanks! As we've removed the CryptoKeyStorage trait in this new version, it makes sense to place everything in here. Note, I'm going to refactor this file in a future PR (see my comment in the PR description).